### ISSUE TITLE
Question about learning performance when finetuning LayoutLM V3 on PubLayNet-like dataset

### ISSUE THREAD

**Issue by naourass** [microsoft/unilm#724]
I'm trying to finetune _LayoutLM V3_ [Base model](https://huggingface.co/microsoft/layoutlmv3-base) using the provided `dit/train_net.py` script on my own custom dataset that is similar to _PubLayNet_. The learning starts well but after reaching the first checkpoint (2000 iterations) and doing the first evaluation the loss starts going up and the accuracy keeps going down.

Someone can point me to some critical factors that may lead to this behavior ? Thanks in advance !

![image](https://user-images.githubusercontent.com/4897498/169405839-cc86e102-6cda-4734-9131-24f26f3777ec.png)
![image](https://user-images.githubusercontent.com/4897498/169405868-31bdc2d4-187f-4d67-a36e-fde5e3f15c47.png)

**Comment by HYPJUDY** [comment]
I am not sure if this problem is related to parameter tuning, dataset characteristics, or pre-trained models. 
We encountered the loss divergence problem with one of our ablated models (see Figure 4 in the LayoutLMv3 paper) but did not encounter this problem with the full model after parameter tuning. 
The loss usually diverges when the learning rate is too large. Maybe you can firstly try to tune the parameters (e.g., learning rate, batch size). 
If you are only concerned with the object detection problem, you can also have a try on [DiT](https://github.com/microsoft/unilm/tree/master/dit), which uses an object detection framework similar to LayoutLMv3.
Good luck with your experiments!

**Comment by naourass** [comment]
Thanks for your reply and for this awesome paper @HYPJUDY ! The problem was in the dataset and specifically in the validation data. I had to add images with instances that are more consistent with the training data and move those who had hard-to-predict instances to the training dataset. I'm getting great performance now. Thanks again for this awesome project !
![image](https://user-images.githubusercontent.com/4897498/169705498-2aa229df-1556-49d0-831c-289769fd9af6.png)

**Comment by HYPJUDY** [comment]
Glad you solved it : D Thank you for sharing your experience with us!
